attrd/legacy.c

```diff
@@ -703,6 +755,13 @@ main(int argc, char **argv)
         return 100;
     }
 
+    the_lrmd = attrd_lrmd_connect(10, lrm_op_callback);
+    if (the_lrmd == NULL) {
+        crm_err("Aborting startup.(LRMD connect)");
+        return 100;
+    } 
+    crm_notice("LRMD connection active");
```

Comment:
As with atomic attrd, I'd rather skip the connect here, and only connect when needed.